.github/renovate.json5

```diff
@@ -20,10 +20,7 @@
     fetchReleaseNotes: true,
   },
 
-  ignoreDeps: [
-    // svg-spritemap-webpack-plugin v4 requires webpack 5 but EDS currently uses webpack 4 for storybook
```

Comment:
:o I didn't know we weren't using the webpack5 builder for storybook. Did we have difficulties switching over or have we just never bothered?

Reply:
No difficulties switching over but EDS is currently bundled via `tsc` and didn't want to include webpack in the deps

Reply:
Should be able to use Webpack 5 without including the dep in the final package, but still may not be worth the effort to bring it in and configure.